src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -73,20 +73,21 @@ const createAppServiceInstalls = async (
 	api: sbvrUtils.PinejsClient,
 	appId: number,
 	deviceIds: number[],
-): Promise<void> =>
-	createReleaseServiceInstalls(api, deviceIds, {
+): Promise<void> => {
+	await createReleaseServiceInstalls(api, deviceIds, {
```

Comment:
This is possibly fixing issues with things being inaccessible for a bit due to permissions on the cloud -- did you observe any yourself?

Reply:
This shouldn't have any functional change, it just improves potential stack traces